[Ignore this] Add the search feature

preciousonah opened the pull request:
**Description**

- Add a search feature that can search for an outfit by keyword 

**Milestones**

- Create new component to add the search feature 
- User can search for an outfit with a keyword
Here's a preview of the website: 
![Image 7-7-23 at 3 49 PM](https://github.com/preciousonah/Fashion-Messaging-Capstone-Project-/assets/121525219/029ae56a-1e4c-4192-babf-0c50e4b0a43f)

preciousonah committed:
Add the search feature

kgoodell commented on fashion-messaging-app-ui/src/components/App/App.jsx:
```diff
@@ -1,39 +1,43 @@
+
   useEffect(() => {
-    clientAPI.photos.search({ query, per_page: PHOTOS })
+    const query = searchQuery || defaultQuery;
```
does this just set query to searchQuery if it exists and defaultQuery if it hasn't been set?

preciousonah replied:
> does this just set query to searchQuery if it exists and defaultQuery if it hasn't been set?
Yes it's a logical operator that I set so that if the user types in a search keyword it the query is assigned to that value but If it returns false then the page just displays the default query which is "Fashion"

kgoodell commented on fashion-messaging-app-ui/src/components/Search/Search.jsx:
```diff
@@ -1,18 +1,32 @@
+        onChange={handleSearchInput}
+        placeholder="Search your outfits!"
+      />
+      <button type="submit"className="submitBtn"><FaSearch/></button>
```
nit: could you add spaces between "submit" and className?

preciousonah replied:
> nit: could you add spaces between "submit" and className?

Okay!

preciousonah committed:
Deploy website

preciousonah committed:
correct a typo

preciousonah committed:
Merge branch 'main' of github.com:preciousonah/Fashion-Messaging-Capstone-Project- into search-function

preciousonah committed:
Updates

preciousonah committed:
Deploying website

preciousonah committed:
Adding dist

preciousonah committed:
Committing changes before subtree add

preciousonah committed:
Add 'dist/' from commit '5c9141c90722b5caa4f1031f450e5c153705ff7c'

git-subtree-dir: dist
git-subtree-mainline: 17e1c353e8ba790c744535b87e4932bc30444a0f
git-subtree-split: 5c9141c90722b5caa4f1031f450e5c153705ff7c

preciousonah committed:
Revert Deploying website
